### ISSUE TITLE
Popped out Sidebar tabs will have unique tab id's

### ISSUE THREAD

**Issue by anathemamask** [foundryvtt/foundryvtt#5506]
When a sidebar tab has been popped out its id field exists both on the popped out version and the original sidebar. It might be worthwhile to append "popped-out-" to the beginning of the id field to resolve potential issues with screen readers (and css selectors)

**Comment by anathemamask** [comment]
> I don't think it's a particular issue for development itself to have duplicate IDs. The following is the line of code from my module Combat Tracker Disposition that finds the rows to colour, which would need altering after these changes.
> 
> ```js
> const combatantRows = document.querySelectorAll(`:is(#combat-tracker, #combat-popout) [data-combatant-id="${combatant.id}"]`);
> ```
> 
> While this could be remedied by pushing the combat tracker and the popout rows to the array separately, that feels unnecessarily complicated. I'd like to be clear in that I have very little understanding of how screenreaders work, but are the combat-tracker/popout ids not sufficient? If not, the a11y benefits in that regard may outweigh the drawbacks in others, but I just wanted to make my voice heard slightly_smiling_face

This is very specifically an accessibility issue as it does not adhere to WCAG standards, so unless it is disruptive to other factors should be done to align with that.
